app/mailers/concerns/ensure_quoted_printable.rb

```diff
@@ -0,0 +1,18 @@
+# Sendgrid enforces line length limits of 998 characters. This
+# is according to the RFC spec: http://tools.ietf.org/html/rfc5322
+# see section 2.1.1 for more info.
+#
+# It will break lines with a new line when the limit is reached,
+# this can break links.
+#
+# This module when included in a Mailer will ensure the
+# Content-Transfer-Encoding header is set to quoted-printable
+# which results in correctly formatted html and text emails.
+
+module EnsureQuotedPrintable
```

Comment:
We no longer need this, because we aren't going to be generating URLs longer than 998 characters.